### PR TITLE
Update image data pipeline tutorial

### PR DESCRIPTION
**Pull request by BurnhamG**
I've made some typo corrections and readability improvements.

### CODE REVIEW

File: site/en/tutorials/load_data/images.ipynb [hunk 1]
```diff
@@ -927,7 +928,7 @@
         "id": "CboYya6LmdQI"
       },
       "source": [
-        "So before the passing it to the MobilNet model, we need to convert the input from a range of `[0,1]` to `[-1,1]`."
+        "So before passing data to the MobileNet model, we need to convert the input from a range of `[0,1]` to `[-1,1]`."
```
Comment (lamberta) [inline site/en/tutorials/load_data/images.ipynb]: Not your doing but since you're in here, can you replace "we" with "you" ?
This is the Google developer docs style: https://developers.google.com/style/person

File: site/en/tutorials/load_data/images.ipynb [hunk 2]
```diff
@@ -993,7 +994,7 @@
         "id": "vrbjEvaC5XmU"
       },
       "source": [
-        "So build a model wrapped around MobileNet, and use `tf.keras.layers.GlobalAveragePooling2D` to average over those space dimensions, before the output `tf.keras.layers.Dense` layer:"
+        "Because of this output shape, build a model wrapped around MobileNet, using `tf.keras.layers.GlobalAveragePooling2D` to average over the space dimensions before the output `tf.keras.layers.Dense` layer:"
```
Comment (lamberta) [inline site/en/tutorials/load_data/images.ipynb]:
```suggestion
        "Because of this output shape, build a model wrapped around MobileNet using `tf.keras.layers.GlobalAveragePooling2D` to average over the space dimensions before the output `tf.keras.layers.Dense` layer:"
```

File: site/en/tutorials/load_data/images.ipynb [hunk 3]
```diff
@@ -840,16 +840,17 @@
         "\n",
         "1. The order is important.\n",
         "\n",
-        "  * A `.shuffle` before a `.repeat` would shuffle items across epoch boundaries (some items will be seen twice before others are seen at all).\n",
+        "  * A `.shuffle` after a `.repeat` would shuffle items across epoch boundaries (some items will be seen twice before others are seen at all).\n",
```
[the remark this answers is not in the document]
Comment (BurnhamG) [reply site/en/tutorials/load_data/images.ipynb]: My understanding from the Data Input Pipeline Performance page was that the `.shuffle` should come before both of those operations, so just changing this one "before" to "after" was my intent. Would you rather I combine the two bullet points into one to remove the repetitive starting phrases?

File: site/en/tutorials/load_data/images.ipynb (hunk 1 above)
Comment (BurnhamG) [reply site/en/tutorials/load_data/images.ipynb]: Thank you for the link, I've updated the notebook accordingly!

File: site/en/tutorials/load_data/images.ipynb (hunk 3 above)
[the remark this answers is not in the document]
Comment (lamberta) [reply site/en/tutorials/load_data/images.ipynb]: Thanks. Will confirm this is the intended behavior and, in that case, can leave as separate bullet points and perhaps emphasize the *after*.
@yashk2810 PTAL.

[the remark this answers is not in the document]
Comment (yashk2810) [reply site/en/tutorials/load_data/images.ipynb]: Shuffling after repeat will shuffle items across epoch boundaries. So this seems to be right.
